packages/paste-core/components/avatar/src/index.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Text} from '@twilio-paste/text';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {IconSize, isIconSizeTokenProp} from '@twilio-paste/style-props';
+import {getComputedTokenNames, getInitialsFromName} from './utils';
+
+export interface AvatarProps extends React.HTMLAttributes<'div'> {
+  name: string;
+  size: IconSize;
+  src?: string;
+}
+const Avatar: React.FC<AvatarProps> = ({name, children, size = 'sizeIcon70', src, ...props}) => {
+  const computedTokenNames = getComputedTokenNames(size);
+  if (src != null && name === undefined) {
+    console.error('Paste Avatar: You must provide a name if you are displaying an image');
+  }
+  return (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      as="div"
+      backgroundColor="colorBackgroundPrimaryLight"
+      borderRadius="borderRadiusCircle"
+      overflow="hidden"
+      size={size}
+    >
+      {src != null ? (
+        <Box {...safelySpreadBoxProps(props)} as="img" alt={name} maxWidth="100%" src={src} size={size} title={name} />
```

Comment:
Does having the `title` and `alt` tags on the image version announce to screen readers two times?

Reply:
It shouldn't do, there is an algorithm for calculating the accessible name of the element. Both alt and title contribute to the accessible name, but alt overrides title in priority. As long as they are the same it's not a problem

Reply:
Got it. Thanks for the clarification.